apps/yggdrasil/lib/yggdrasil/role_permission.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.RolePermission do
+  use Ecto.Schema
+
+  @primary_key false
+  schema "roles_permissions" do
+    field :resource, :string
+    field :permission, :string
```

Comment:
Yeah that's what I was thinking, it reads better for now and we may never need more and if we do should be simple to change.

I will say that I'm starting to dig Ecto, there are some rough spots but overall it's really nice to use. I haven't really had to fight it and with ecto 2.0 I think it will be even better so. You can really bend this to your will I'm finding out which really lets you sort of just design and it doesn't get in your way with too many opinions about how things "should" work.